src/adapters/analytics/ga.js

```diff
@@ -56,7 +55,8 @@ exports.enableAnalytics = function ({ provider, options }) {
       sendBidResponseToGa(bid);
 
     } else if (eventObj.eventType === BID_TIMEOUT) {
-      _timedOutBidders = args.bidderCode;
+      const bidderArray = args;
```

Comment:
The same can be said for the other events where the code is using the `bid` var, but that's an anti-pattern. 😉